components/style/values/specified/box.rs

```diff
@@ -130,3 +131,49 @@ define_css_keyword_enum! { OverflowClipBox:
     "content-box" => ContentBox,
 }
 add_impls_for_keyword_enum!(OverflowClipBox);
+
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue, ToCss)]
+/// Provides a rendering hint to the user agent,
+/// stating what kinds of changes the author expects
+/// to perform on the element
+///
+/// <https://drafts.csswg.org/css-will-change/#will-change>
+pub enum WillChange {
+    /// Expresses no particular intent
+    Auto,
+    #[css(comma, function, iterable)]
```

Comment:
This shouldn't have `function`, otherwise it'd serialize as `animatable-features(foo, bar)` instead of just `foo, bar`.